app/templates/base/home.html

```diff
@@ -5,16 +5,133 @@
 
 {% block main_column %}
 
+  <ul class="tabs">
+    {% if current_user.is_superuser %}
+      <li>Superuser</li>
+    {% endif %}
+    <li>Analytical tools</li>
+    <li>Data</li>
+    <li>Webapps</li>
+  </ul>
+
   {% if current_user.is_superuser %}
-    <h2 class="heading-medium">Superuser functions</h2>
-    <ul class="list list-bullet">
-      <li><a href="{{ url_for('users.list') }}">List all users</a></li>
-      <li><a href="{{ url_for('apps.list') }}">List all apps</a></li>
-      <li><a href="{{ url_for('buckets.list') }}">List all app data sources</a></li>
-    </ul>
+    <section class="tab-panel">
+      <h2 class="heading-medium">Superuser functions</h2>
+      <ul class="list list-bullet">
+        <li><a href="{{ url_for('users.list') }}">List all users</a></li>
+        <li><a href="{{ url_for('apps.list') }}">List all apps</a></li>
+        <li><a href="{{ url_for('buckets.list') }}">List all app data sources</a></li>
+      </ul>
+    </section>
   {% endif %}
 
-  {% include 'users/includes/tables.html' %}
+  <section class="tab-panel">
+    <h2 class="heading-medium">Your tools</h2>
+    <a href="{{ url_for('tools.list') }}" class="button button-secondary">View your tools</a>
+  </section>
+
+  <section class="tab-panel">
+    <div class="form-group">
+      <h2 class="heading-medium">{{ heading_prefix }} app data sources</h2>
+      {% if user.users3buckets.length %}
+        <table class="form-group">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>User has admin access</th>
+              <th>User has read/write access</th>
+              <td><span class="visuallyhidden">Actions</span></td>
+            </tr>
+          </thead>
+          {% for users3bucket in user.users3buckets %}
+            <tr>
+              <td><a href="{{ url_for('buckets.details', {id: users3bucket.id}) }}">{{ users3bucket.s3bucket.name }}</a></td>
+              <td>
+                {% if users3bucket.is_admin %}
+                  Yes
+                {% else %}
+                  No
+                {% endif %}
+              </td>
+              <td>
+                {% if users3bucket.access_level == 'readonly' %}
```

Comment:
You may want to use `{{ yes_no(users3bucket.access_level, 'readwrite') }}`.

Reply:
👍